sklearn/tree/_classes.py

```diff
@@ -1419,7 +1439,7 @@ class ExtraTreeClassifier(DecisionTreeClassifier):
         the input samples) required to be at a leaf node. Samples have
         equal weight when sample_weight is not provided.
 
-    max_features : int, float, {"auto", "sqrt", "log2"} or None, default="auto"
+    max_features : int, float, {"auto", "sqrt", "log2"} or None, default="sqrt"
```

Comment:
I like the idea of changing the default directly, since the "auto" and "sqrt" are the same.